exercises/largest-series-product/src/main/java/LargestSeriesProductCalculator.java

```diff
@@ -1,5 +1,11 @@
 public class LargestSeriesProductCalculator {
```

Comment:
Both the class and the methods can be made [`package private`](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html). The class's access modifier was there from before so that's not at all your fault :) But it would be a nice improvement to make while we're modifying the class. It's generally best practice to be as restrictive as possible with access modifiers :)